lib/connection_manager.go

```diff
@@ -35,24 +33,10 @@ type ConnectionManager struct {
 	// doesn't need a reference to the Server object. But for now we keep things lazy.
 	srv *Server
 
-	// When --connectips is set, we don't connect to anything from the addrmgr.
-	connectIps []string
-
-	// The address manager keeps track of peer addresses we're aware of. When
-	// we need to connect to a new outbound peer, it chooses one of the addresses
-	// it's aware of at random and provides it to us.
-	AddrMgr *addrmgr.AddrManager
```

Comment:
Can you leave these fields here and instead pass them through or read them from the ConnectionController? Again, a good goal here is to reduce the amount of code that's simply moved around to streamline downstream reviews